### DISCUSSION TITLE
Limitänderung per POST Request klappt nicht

### DISCUSSION THREAD

**Discussion by 2000ede** [tbnobody/OpenDTU#1687] [Q&A]
Hallo,

wenn ich von außen das Limit ändere über einen POST Befehl, klappt das mit einem System von mir, mit dem anderen nicht.
Bekommt man irgendwie ein Logging?

links geht, rechts gehts nicht. liegt es am Encoding? (die Seriennummern nur Testnummern, an ihnen liegt es nicht)
Bin nur Anfänger!

![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/14c1f6bc-06c4-4b97-851c-8c9c1274d131)

**Comment by stefan123t** [comment]
Hallo @2000ede,
rechts verwendest Du bzw der Client/Browser ein sog chunked encoding.

Ich weiss nicht ob der eingesetzte ESP AsyncWebserver in der OpenDTU diese Salami-Taktik auch mit unterstützt ?

Womit wird denn der rechte Aufruf durchgeführt bzw kann man die wenigen Daten nicht einfach ohne chunked encoding senden ?

@tbnobody müssen wir chunked encoding und ggf andere Anliegen der Clients explizit zurückweisen ?

**Reply by 2000ede** [reply]
Hi Stefan,
danke für deine Antwort. Ich bin nur hartnäckiger try&error-Einsetzer. Hab mir mit tas mota an visualisierung schon einiges umgesetzt. nun will ich mit dem system, was auch meinen stromzähler auswertet, direkt opendtu versorgen. aktuell mache ich das wie gesagt über google script - funktioniert auch super. aber wollte es eigentlich komplett ohne cloud probieren.

**Reply by StefanOberhumer** [reply]
Accept-Encoding: identity;q=1.0, chunked;q=0.1, *;q=0.1
heißt, dass die bevorzugte(Wert 1.0) Encodingart [identity](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding#identity) (= nichts ändern) ist.

Ich glaube somit dass es nicht an dem "Accept-Encoding" header liegt !

**Reply by 2000ede** [reply]
ich kann ich es weiter eingrenzen?
ich habe mittlerweile 3 Entwickerkollegen, die sich die Zähne daran ausbeissen

**Comment by tbnobody** [comment]
Wie schauen denn die return header und die response aus?

**Reply by 2000ede** [reply]
Ich bin zu wenig Entwickler. Habe nun Workaround über berry, da klappt es wohl. 
Ich verfolge es nach zig Anläufen hier erstmal nicht mehr weiter.
